src/plugins/1/strategies/index.md

```diff
@@ -0,0 +1,5 @@
+---
+layout: full.html.hbs
+title: Strategies
+order: 500
+---
```

Comment:
Empty content.

Reply:
I have to have this empty `index.md`, otherwise Metalsmith won't display the Strategie menu and won't generate its sub-menus.
Or i didn't get what you meant ?